firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -1134,9 +1136,11 @@ private void validateActiveTargets() {
       TargetData expectedTarget = expectedQueries.get(0);
       TargetData actualTarget = actualTargets.get(expected.getKey());
 
-      // TODO: validate the purpose of the target once it's possible to encode that in the
-      // spec tests. For now, only validate properties that can be validated.
+      // TODO: Simplify the TargetData validation when the sequenceNumber can be consistent as well.
```

Comment:
I think this comment should be re-worded a bit. I don't fully understand what it means. Does something like this make sense:

TODO: Replace the assertEquals() checks on the individual properties of TargetData below with the single assertEquals on the TargetData objects themselves if the sequenceNumber is ever made to be consistent.